source/server/connection_handler_impl.h

```diff
@@ -192,6 +195,8 @@ class ConnectionHandlerImpl : public Network::ConnectionHandler,
     ActiveConnections& active_connections_;
     Network::ConnectionPtr connection_;
     Stats::TimespanPtr conn_length_;
+    Network::ListenerConfig& config_;
+    std::unique_ptr<StreamInfo::StreamInfo> stream_info_;
```

Comment:
I would like to capture missing filter chains as well, for which connections don't exist. Hence, usin this somewhat awkward ownership.